_posts/2021-08-23-gsoc-2021-project.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: Google Summer of Code 2021 - Dask Project
+author: Freyam Mehta
+theme: twitter
+---
+
+{% include JB/setup %}
+
+This blog covers the work done by me during the Google Summer of Code 2021. Dask took part in the program under the NumFOCUS umbrella organization.
+
+Google Summer of Code is a global program focused on bringing more student developers into open source software development. Students work with an open source organization on a 10 week programming project during their break from school.
```

Comment:
Is this sentence from the GSOC website? If so, we need to put it in quotation marks.

Reply:
(Removed the sentence)